mlops_utilities/actions.py

```diff
@@ -119,7 +119,7 @@ def run_pipeline(
 
 
 def deploy_model(
-    sagemaker_session: PipelineSession,
+    sagemaker_pipeline_session: PipelineSession,
```

Comment:
@leonid133, I believe we should not switch to `PipelineSession` here; we can keep a regular `Session` object. The only function that is supposed to be updated is `upsert_pipeline` 🙂